content/docs/examples/multicluster/gateways/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Gateway-connected Clusters
```

Comment:
connected -> Connected

Reply:
Even though it's hyphenated (`Gateway-Connected Clusters`)? Would that apply to [this one](https://preliminary.istio.io/docs/examples/multicluster/split-horizon-eds/) too (`Cluster-Aware`)?

Reply:
I'm not a native English speaker, so this particular grammatical subtlety is beyond my skill set.

There's got to be a capitalization rule for this...

Reply:
According to my quick google search, I think the answer is yes, capitalize it.